DigitalLearningSolutions.Web/Controllers/LoginController.cs

```diff
@@ -175,6 +175,16 @@ UserAccountSet accounts
                 IsPersistent = rememberMe,
                 IssuedUtc = DateTime.UtcNow
             };
+
+
+            /* Course progress doesn't get updated if the auth token expires by the end of the tutorials as some

```

Comment:
"as" is not quite right here. Better to have "... tutorials. Some...". There's no causal relation between those two things. Looks good other than that.